styles/styles.css

```diff
@@ -700,6 +700,209 @@ main > .section > .grid-container {
   }
 }
 
+.util-listing-cta {
+  background: var(--color-background-highlighted-element);
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  border-radius: 1.5rem 1.5rem 0 0;
+  padding: 1.5rem 1.25rem 2rem;
+  display: flex;
+  flex-direction: column;
+  gap: 1.5rem;
+  box-shadow: var(--drop-shadow-emphasized);
+
+  @media (min-width: 36rem) {
+    padding: 1.5rem 2rem 2rem;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  @media (min-width: 80rem) {
+    display: none;
+  }
+
+  span {
+    color: var(--color-text);
+
+    @media (min-width: 36rem) {
+      font-size: var(--spectrum-font-size-700);
+    }
+  }
+  
+  .button {
+    @media (min-width: 36rem) {
+      width: auto;
+    }
+  }
+}
+
+.util-listing-container {
+  @media (min-width: 80rem) {
+    display: grid;
+    grid-template-columns: 8fr 1fr 3fr;
+    grid-template-rows: auto 1fr;
+    grid-template-areas:
+      "title title title"
+      "content . details";
+  }
+
+  > h1 {
+    margin-block-start: 2rem;
+    margin-block-end: 1rem;
+    
+    @media (min-width: 36rem) {
+      margin-block-start: 4rem;
+      margin-block-end: 2.5rem;
+    }
+    
+    @media (min-width: 80rem) {
+      margin-block-start: 12rem;
+      margin-block-end: 10rem;
+      grid-area: title;
+    }
+
+    @media (min-width: 105rem) {
+      margin-block-end: 18rem;
+    }
+  }
+
+  > .util-listing-details {
+    @media (min-width: 80rem) {
+      grid-area: details;
+    }
+  }
+
+  > .util-listing-content {
+    > h2:first-of-type {
+      margin-block-start: 0;
+    }
+
+    @media (min-width: 80rem) {
+      grid-area: content;
+    }
+  }
+}
+
+.util-listing-details {
+  display: grid;
+  gap: 1rem;
+  grid-template-areas:
+  "button button"
+  "location location"
+  "detail-grid detail-grid";
+  
+  @media (min-width: 36rem) {
+    grid-template-columns: 1fr 2fr auto;
+    grid-template-areas:
+    "location detail-grid button";
+  }
+  
+  @media (min-width: 80rem) {
+    align-self: start;
+    position: sticky;
+    top: 4rem;
```

Comment:
I sent you a slack message about an improvement, but lets use the `--page-nav-height` property here.

Reply:
I added 1rem as well so it's not bumping up against the nav.